view/partial/tasks.html

```diff
@@ -57,10 +57,10 @@
 				<div class="btn-group options-button text-right">
 					<button type="button" class="btn btn-info btn-xs dropdown-toggle" data-toggle="dropdown"><span class="sr-only">Options</span><span class="glyphicon glyphicon-cog"></span></button>
 					<ul class="dropdown-menu pull-right" role="menu">
-						<li><a href="{{href}}/edit">Edit this task</a></li>
-						<li><a href="{{href}}/delete">Delete this task</a></li>
+						<li role="presentation"><a href="{{href}}/edit" role="menuitem">Edit this task</a></li>
```

Comment:
same comment here as with the `menu` stuff above

Reply:
Thank you @hollsk for the insights on ARIA roles, I removed a few more that were unnecessary IMO, have a look!